data/hai/hai missions.txt

```diff
@@ -209,6 +209,77 @@ mission "Ask the Hai about the Unfettered"
+		set "warned about hai merchant politics"
+		conversation
+			`As you sit in a spaceport bar waiting for the food that you've ordered, a Hai unceremoniously pulls up a chair opposite you and takes a seat. He looks to be a merchant, and one who has seen more than a few things besides. He squints at you before speaking, "You are Captain <last>, are you not?"`
+			choice
+				`	"That would be me."`
+					goto acknowledge
+				`	"I think you have the wrong captain."`
+			`	He slaps a device on the table, and a small holographic projection of your face springs up and begins rotating around. "No, I think not."`
+			label acknowledge
+			`	He heaves a sigh. "You've been fighting the Unfettered, you see, and there are some Hai that don't take too kindly to that. They view you as an outsider sticking your nose into Hai internal politics. Some of those are what we merchants like to call Sympathizers, and they can get a little carried away with themselves and fire off a few shots if they see someone like yourself getting a little too gun-happy where the Unfettered are concerned."`
+			`	With a wave of his hand he indicates the spaceport around you. "You might have potentially noticed some difficulty landing. You see, because there are lots of people who don't like outsiders putting their noses in where they're not wanted, if you happen to shoot back and accidentally destroy a Sympathizer's ship, the authorities don't much care who shot first. There are enough of those opinions to ensure that if you shoot down a Sympathizer, it is treated no differently than if you shot down any other Hai merchant. The security forces will treat it the same.`
+			`	"Now the merchants, we're a different matter. It is a topic of great and ongoing debate among us, and some people are less offended when you shoot down someone they've been arguing with. Everyone has friends on all sides though, so you should watch yourself nonetheless. Sympathizers may become upset enough to shoot at you anywhere they see you, but the simple fact of the matter is that, if you retaliate against a Hai, any Hai, it'll upset someone and count against you. Even if that Hai was a Sympathizer, and even if they shot first."`
+			`	He looks at you very seriously to make sure what he's said has sunk in.`
+			branch "low reputation"
+				"reputation: Hai" < 1
+			choice
+				`	"I understand, thank you for warning me."`
+					goto end
+				`	"Why should I get in trouble for defending myself against Sympathizers?"`
+					goto defending
+			label "low reputation"
+			choice
+				`	"I understand that I've stepped on some toes."`
+					goto trouble
+				`	"How could I be in trouble for defending myself against Sympathizers?"`
+			label defending
+			action
+				set "Hai Sympathizer Warning: Explained Trouble"
```

Comment:
This should probably be lower case.
Mission autoconditions (or conditions meant to look like them) are capitalised, but others almost always aren't.
I know you're using the mission name here, but it's not a mission autocondition.

Reply:
While I'm aware that's generally true, a ways back when `to display` was implemented people started using them capitalised where they referred to the mission name and were used only in the mission (vs conditions set for later missions).
e.g.
```
			choice
				`	"The Hai are no longer a secret in human space, but they are concerned how you ended up here."`
					goto revealed
					to display
						not "Hai Reveal [C02] Cleanup: label revealed"
				`	"What happened to the ship that brought you here?"`
					goto kidnappers
					to display
						not "Hai Reveal [C02] Cleanup: label kidnappers"
				`	"Well, let's get you home before anything else happens."`
					goto depart
			label revealed
			action
				set "Hai Reveal [C02] Cleanup: label revealed"`
```
I'm mostly familiar with it having been applied in HR, because that's where I see it.
Having a look around it appears that:
- HR uses capitals.
- Remnant stuff uses lowercase.
- PR'd Korath stuff uses capitals.
- Unfettered PR is currently a mess mixing both capitals and lowercase sometimes within the same condition.

I do kind of like them being capitalised, because it means they show up next to the mission in a save file which makes troubleshooting things easier, but I'm not pressed either way.
Seems like we've got a conflict of convention at present though.

Reply:
Note: There were at least half a dozen missions rejigged for HR using this before I even learned how to use the syntax for this - this usage didn't originate from me.

Reply:
I thought I was using lower case but I must have slipped then
I used "unfettered: smth" as convention instead of basing them on missions most of the time

Reply:
I'd prefer all lowercase. Looking to https://github.com/endless-sky/endless-sky/pull/7660 where a large number of to display nodes were added, lowercase condition names are used. The condition names are also abbreviated, not being the full name of the mission that the conditions are being used in (such as "saes" for "Sheragi Archaeology: Emerald Sword").

Reply:
Using the PR of the person posing the possibility as evidence to back up the decision to do so isn't really the most compelling argument.
Is there a tangible benefit that can be defined for using lowercase/abbreviated `to display` conditions?

I would like to know if there's a positive rationale for it, since I'll likely have to go through and standardise them all for HR if this is the direction we take.

Reply:
Manually set conditions. regardless of location, are generally lowercase. This allows mission conditions to be separated from all other conditions in the save file, since the save file conditions are sorted alphabetically with uppercase first.

Reply:
These conditions should have been unset at the end of the conversations anyway since they are unlikely to be used later on. Probably we should start doing that now before it gets too out-of-hand in the future.

Reply:
Ah, that's a thought. Is there a simple way to unset all conditions regardless of if used? I was under the impression that just doing `set 'condition' = 0` will leave it as still being there because it's now got an assigned value.

Reply:
clear "condition" I think